composer.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "josegonzalez/cakephp-version",
-    "description": "CakePHP plugin to allow versioning of records",
-    "type": "cakephp-plugin",
-    "keywords": ["cakephp", "revision", "version"],
+    "description": "Cake orm behavior to allow versioning of records",
+    "type": "library",
```

Comment:
this was what I was referring in the comment https://github.com/josegonzalez/cakephp-version/issues/8#issuecomment-171860536 .

I am wondering if you have a type of `cakephp-plugin` how will it then handle the same? I will try and see how it will work.